src/journal-remote/journal-remote-write.c

```diff
@@ -33,6 +39,17 @@ Writer* writer_new(RemoteServer *server) {
         w->n_ref = 1;
         w->server = server;
 
+        if (is_dir(server->output, true) > 0)
```

Comment:
Here, please also comment the boolean: `/* follow = */ true`

Reply:
done